source/d3d11/state_tracking.hpp

```diff
@@ -49,6 +50,11 @@ namespace reshade::d3d11
 		bool _has_indirect_drawcalls = false;
 		std::unordered_map<com_ptr<ID3D11Texture2D>, depthstencil_info> _counters_per_used_depth_texture;
 #endif
+
+	private:
+#if RESHADE_DEPTH
+		std::unordered_map<size_t, int> _shown_count_per_depthstencil_address;
```

Comment:
Would be easier to make this a `std::unordered_map<ID3D11Resource *, unsigned int>`, to avoid the ugly address casts. Without the use of `com_ptr` this won't reference count the object, so is otherwise identical. Same applies to D3D12 too.

Reply:
Good point. I feared it might refer to a texture that might have been removed and it would crash but we don't use the actual pointer in that fashion so that's indeed doable. 
Changed.